tds-itemselection-impl/pom.xml

```diff
@@ -39,6 +39,18 @@
 			<artifactId>tds-dll-mysql</artifactId>
 			<version>0.0.1-SNAPSHOT</version>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.11</version>
+            <scope>test</scope>
+        </dependency>
+		<dependency>
```

Comment:
Seems like it's Github 
![image](https://cloud.githubusercontent.com/assets/1069704/20067251/4438f3a2-a4c9-11e6-95bc-2c4bceb4e187.png)